systemtest/src/test/java/io/strimzi/systemtest/metrics/MetricsST.java

```diff
@@ -167,7 +166,7 @@ void testKafkaExporterDataAfterExchange() {
             assertThat("Value from collected metric should be non-empty", !value.isEmpty());
             assertThat("Metrics doesn't contain specific values", value.contains("kafka_consumergroup_current_offset"));
             assertThat("Metrics doesn't contain specific values", value.contains("kafka_consumergroup_lag"));
-            assertThat("Metrics doesn't contain specific values", value.contains("kafka_topic_partitions{topic=\"" + TEST_TOPIC_NAME + "\"} 7"));
+            assertThat("Metrics doesn't contain specific values", value.contains("kafka_topic_partitions{topic=\"" + TOPIC_NAME + "\"} 7"));
```

Comment:
I already fixed this in my PR - by different way. But I think it will be better to have it all on one place

Reply:
We can solve conflict when it appears